Flask/microservices/distance-service/app.py

```diff
@@ -0,0 +1,7 @@
+from website import app
+import static.config as config
+
+app = app
+
+if __name__ == '__main__':
+    app.run(host=config.HOST, port=config.PORT)

```

Comment:
When i start the application i get this message:
```
WARNING: This is a development server. Do not use it in a production deployment.
Use a production WSGI server instead
WARNING: This is a development server. Do not use it in a production deployment.
```
This can make huge impact on performance.
It happens on this service and in place-service